package.json

```diff
@@ -34,30 +34,30 @@
     "sass": "1.55.x",
     "svelte": "^3.x"
   },
+  "dependencies": {
+    "svelte": "^3.x",
+    "material-components-web": "^14.0.0"
+  },
   "devDependencies": {
-    "@semantic-release/changelog": "^6.0.2",
-    "@semantic-release/git": "^10.0.1",
     "@storybook/addon-actions": "^6.5.16",
     "@storybook/addon-docs": "^6.5.16",
     "@storybook/addon-essentials": "^6.5.16",
     "@storybook/addon-links": "^6.5.16",
-    "@storybook/addon-postcss": "^2.0.0",
     "@storybook/addon-svelte-csf": "^2.0.11",
     "@storybook/builder-webpack5": "^6.5.16",
     "@storybook/manager-webpack5": "^6.5.16",
+    "@storybook/addon-postcss": "^2.0.0",
     "@storybook/preset-scss": "^1.0.3",
     "@storybook/svelte": "^6.5.16",
+    "@semantic-release/changelog": "^6.0.2",
+    "@semantic-release/git": "^10.0.1",
     "conventional-changelog": "^3.1.25",
```

Comment:
I'm curious why these three were moved (out of alphabetical order). Not a big deal, just seemed unexpected.

Reply:
I think it happened when I was moving stuff in and out of devDependencies and I don't have prettier or whatever doing it for me. It might be better to do in another PR since those aren't the only ones out of order.